generate.go

```diff
@@ -49,14 +49,14 @@ func (zp *ZoneParser) generate(l lex) (RR, bool) {
 	if err != nil {
 		return zp.setParseError("bad stop in $GENERATE range", l)
 	}
-	if end < 0 || start < 0 || end < start {
+	if end < 0 || start < 0 || end < start || (end-start)/step > 65535 {
```

Comment:
See my comment in https://github.com/miekg/dns/issues/1019#issuecomment-537989190 to align this with BIND behaviour. TL;DR both start and stop must be positive integers between 0 and (2^31)-1